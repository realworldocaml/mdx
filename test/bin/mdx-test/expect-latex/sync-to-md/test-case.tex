Mdx can also understand ocaml code blocks:


% $MDX file=sync_to_md.ml,part=toto
\begin{ocaml}
# let x = 3;;
val x : int = 3
# let y = 4;;
val y : int = 4
# print_int x;;
3
- : unit = ()
# print_int y;;
4
- : unit = ()
\end{ocaml}

% $MDX file=sync_to_md.ml,part=zzz
\begin{ocaml}
\end{ocaml}

% $MDX file=sync_to_md.ml,part=42
\begin{ocaml}
\end{ocaml}

% $MDX file=sync_to_md.ml,part=
\begin{ocaml}
\end{ocaml}

% $MDX file=sync_to_md.ml
\begin{ocaml}
\end{ocaml}

\begin{ocaml}
# let x = 2;;
val x : int = 2
# print_int x;;
3
- : unit = ()
\end{ocaml}
