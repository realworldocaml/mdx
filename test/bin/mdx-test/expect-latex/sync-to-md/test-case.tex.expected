Mdx can also understand ocaml code blocks:


% $MDX file=sync_to_md.ml,part=toto
\begin{ocaml}
let x = 34
let f = 42.3
let s = "toto"
let f x u = u x

let () =
  print_int x;
  print_float f
;;
\end{ocaml}

% $MDX file=sync_to_md.ml,part=zzz
\begin{ocaml}
let () =
  print_string s
;;
\end{ocaml}

% $MDX file=sync_to_md.ml,part=42
\begin{ocaml}
let () =
  f x print_int
\end{ocaml}

% $MDX file=sync_to_md.ml,part=
\begin{ocaml}
let () =
  ();
  ()
;;
\end{ocaml}

% $MDX file=sync_to_md.ml
\begin{ocaml}
let () =
  ();
  ()
;;

let x = 34
let f = 42.3
let s = "toto"
let f x u = u x

let () =
  print_int x;
  print_float f
;;

let () =
  print_string s
;;

let () =
  f x print_int
\end{ocaml}

\begin{ocaml}
# let x = 2;;
val x : int = 2
# print_int x;;
2
- : unit = ()
\end{ocaml}
